src/styles/utils/_reset.scss

```
@@ -0,0 +1,13 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+
+  h2,
+  p {
+    margin: 0;
+  }
+}
```

Move font-family to the html selector
```suggestion
html {
  font-family: Roboto, sans-serif;
}

body {
  margin: 0;
}

h2,
p {
  margin: 0;
}
```